cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -484,6 +576,498 @@ def test_frames_outside_are_not_generated(self):
             self.assertTrue(frame.frame in range(6, 10))
         self.assertEqual(i + 1, 4)
 
+    def test_api_v1_tasks_annotations_dump_and_upload_with_datumaro(self):
+        test_name = self._testMethodName
+        # get formats
+        dump_formats = dm.views.get_export_formats()
+
+        for dump_format in dump_formats:
+            if dump_format.ENABLED:
+                dump_format_name = dump_format.DISPLAY_NAME
+
+                # TODO skip failed formats
+                if dump_format_name in [
```

Comment:
> changed points values

Points in segmentation formats (CamVid, Segmentation mask, Mots) can't be stored exactly, because they need to be recognized after converting into masks.

Reply:
OK. These formats are not checked.